src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: $card-width;
+  background: $white-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__photo {
+    height: 134px;
+    width: 160px;
+    align-self: center;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-size: $m-font-size;
+    font-weight: $m-font-weight;
+    line-height: 18px;
+    color: $main-accent-color;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    line-height: 14px;
+    margin-bottom: 16px;
+    font-size: $s-font-size;
+    color: $secondary-color;
+  }
+
+  &__score {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: $s-font-size;
+    line-height: 14px;
+    color: $main-accent-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-size: $m-font-size;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__price-cost {
+    font-size: $xl-font-size;
+    font-weight: $l-font-weight;
+    line-height: 14px;
+    color: $main-accent-color;
+  }
+
+  &__button-buy {
+    width: 166px;
+    background-color: $blue-accent-color;
+    border-radius: 5px;
+    border: 1px solid transparent;
+
+    font-size: $l-font-size;
+    line-height: 16px;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $white-color;
+    font-weight: $l-font-weight;
+    padding: 11px 0;
+
+    transition: color 0.5s ease-in-out, background-color 0.5s ease-in-out;
```

Comment:
Would be nice to create a variable for duration, then we can use it for every transition to make it's duration consistent with other page transitions